boards/arm/mimxrt1062_fmurt6/Kconfig.defconfig

```diff
@@ -0,0 +1,34 @@
+# MIMXRT1062-FMURT6 board
+
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2023 NXP
+
+if BOARD_MIMXRT1062_FMURT6
+
+config BOARD
+	default "mimxrt1062_fmurt6" if BOARD_MIMXRT1062_FMURT6
+
+choice CODE_LOCATION
+	default CODE_FLEXSPI
+endchoice
+
+if FLASH
+
+config FLASH_MCUX_FLEXSPI_HYPERFLASH
+	default y if BOARD_MIMXRT1062_FMURT6
+
+config FLASH_MCUX_FLEXSPI_NOR
+	default n if BOARD_MIMXRT1062_FMURT6
```

Comment:
I don't believe you should need `default n` here, as the driver would not be enabled by default?

Reply:
@danieldegrasse .. This board has a flexspi hyperflash part S26KS512SDPBHI020 so I have made -
FLASH_MCUX_FLEXSPI_HYPERFLASH as default y and 
FLASH_MCUX_FLEXSPI_NOR as default n
![image](https://github.com/zephyrproject-rtos/zephyr/assets/107573027/56ea7eef-81c3-4b00-92a3-69fc899f07a1)

Let me know of your thoughts..

Reply:
It looks like you have removed both in the latest push- is that correct? `CONFIG_FLASH_MCUX_FLEXSPI_HYPERFLASH` should be automatically enabled when `CONFIG_FLASH=y`, based on the compatible of the flash device in your board devicetree

Reply:
Yes, after verifying that CONFIG_FLASH_MCUX_FLEXSPI_HYPERFLASH is enabled based on the compatible field I removed both..

Reply:
Made the changes as suggested